op-chain-ops/upgrades/l1.go

```diff
@@ -310,6 +478,76 @@ func SystemConfig(batch *safe.Batch, implementations superchain.ImplementationLi
 		return err
 	}
 
+	// 2 Step Upgrade
+	{
+		storageSetterABI, err := bindings.StorageSetterMetaData.GetAbi()
+		if err != nil {
+			return err
+		}
+
+		input := []bindings.StorageSetterSlot{
+			// https://github.com/ethereum-optimism/optimism/blob/86a96023ffd04d119296dff095d02fff79fa15de/packages/contracts-bedrock/.storage-layout#L82-L83
+			{
+				Key:   common.Hash{},
+				Value: common.Hash{},
+			},
+			// https://github.com/ethereum-optimism/optimism/blob/86a96023ffd04d119296dff095d02fff79fa15de/packages/contracts-bedrock/.storage-layout#L92
+			{
+				Key:   common.Hash{31: 106},
+				Value: common.Hash{},
+			},
+			// bytes32 public constant L1_CROSS_DOMAIN_MESSENGER_SLOT = bytes32(uint256(keccak256("systemconfig.l1crossdomainmessenger")) - 1);
+			{
+				Key:   common.HexToHash("0x383f291819e6d54073bc9a648251d97421076bdd101933c0c022219ce9580636"),
+				Value: common.Hash{},
+			},
```

Comment:
nit: really should simply compute these in Go, to minimize fat-finger errors.

Reply:
I agree, we should and will improve this in the main op-upgrade tool, but for now I'm going to resolve this as 'wontfix' and I'm not worried about fat fingers because of the state diff checking processes we have in place.